sdk/communication/Azure.Communication.Identity/src/Models/CommunicationUserDetail.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Communication.Identity.Models
+{
+    /// <summary>
+    /// Represents the details of a Communication User.
+    /// </summary>
+    public class CommunicationUserDetail
```

Comment:
Instead of creating a new model we could also rename `CommunicationIdentity` to `CommunicationUserDetail`

Reply:
Although ... we're mapping the flat string to identifier, then we would need more customization in the partial custom model for it. Should be okay to keep it like it is.